### PR TITLE
Integration/0.21.1.2

### PR DESCRIPTION
**Pull request by chrisguida**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,27 +1,27 @@
...
+	DOCKER_CLI_EXPERIMENTAL=enabled docker buildx build --tag start9/bitcoind/main:$(VERSION) --build-arg BITCOIN_VERSION=$(VERSION_STRIPPED) --build-arg N_PROC=$(shell nproc) --platform=linux/arm64 -o type=docker,dest=image.tar .
 
-manager/target/armv7-unknown-linux-musleabihf/release/bitcoind-manager: $(MANAGER_SRC)
-	docker run --rm -it -v ~/.cargo/registry:/root/.cargo/registry -v "$(shell pwd)"/manager:/home/rust/src start9/rust-musl-cross:armv7-musleabihf cargo build --release
-	docker run --rm -it -v ~/.cargo/registry:/root/.cargo/registry -v "$(shell pwd)"/manager:/home/rust/src start9/rust-musl-cross:armv7-musleabihf musl-strip target/armv7-unknown-linux-musleabihf/release/bitcoind-manager
+manager/target/aarch64-unknown-linux-musl/release/bitcoind-manager: $(MANAGER_SRC)
+	docker run --rm -it -v ~/.cargo/registry:/root/.cargo/registry -v "$(shell pwd)"/manager:/home/rust/src start9/rust-musl-cross:aarch64-musl cargo build --release
+	docker run --rm -it -v ~/.cargo/registry:/root/.cargo/registry -v "$(shell pwd)"/manager:/home/rust/src start9/rust-musl-cross:aarch64-musl musl-strip target/aarch64-unknown-linux-musl/release/bitcoind-manager
```
Comment (elvece) [inline Makefile:27]:
```suggestion
```

**Review by elvece — APPROVED** [review]

File: Makefile (hunk 1 above)
Comment (chrisguida) [reply Makefile]: Are we sure this is unnecessary? Doesn't this make the binary smaller?

Comment (elvece) [reply Makefile]: It does but aiden said it was unnecessary

**Review by ProofOfKeags — APPROVED** [review]

**Comment by ProofOfKeags** [comment]
Send it.

**Pull request merged**